chainerx_cc/chainerx/python/routines.cc

```diff
@@ -1002,9 +1003,16 @@ void InitChainerxPooling(pybind11::module& m) {
           py::arg("pad_mode") = "ignore");
 }
 
+void InitChainerxActivation(pybind11::module& m) {
+    m.def("clipped_relu", [](const ArrayBodyPtr& x, Scalar z) { return MoveArrayBody(ClippedRelu(Array{x}, z)); }, py::arg("x"), py::arg("z") = 20.0);
+    m.def("crelu", [](const ArrayBodyPtr& x, nonstd::optional<int8_t> axis) { return MoveArrayBody(Crelu(Array{x}, axis)); }, py::arg("x"), py::arg("axis") = 1);
```

Comment:
Chainer's `F.crelu` does not work for `axis=None` so we can make the `axis` argument here a regular `int8_t` without being an optional. Let me know if I'm wrong.